modules/proxy/mod_proxy_hcheck.c

```diff
@@ -223,6 +229,19 @@ static const char *set_worker_hc_param(apr_pool_t *p,
             PROXY_STRNCPY(worker->s->hcexpr, val);
         } else {
             temp->hcexpr = apr_pstrdup(p, val);
+        } 
+    } else if(!strcasecmp(key, "hctimeout")) {
```

Comment:
Code style (and above/below code) suggests that `} else if () {` should rather be:
```
     }
     else if (...) {
         ...
     }
```
here.